lib/admin/admin.types.ts

```diff
@@ -1,12 +1,12 @@
 export interface AdminObject {
-  type: "admin",
-  id: string,
-  name: string,
-  email: string,
-  job_title: string,
-  away_mode_enabled: boolean,
-  away_mode_reassign: boolean,
-  has_inbox_seat: boolean,
-  team_ids: Array<number>,
-  avatar: string,
+    type: 'admin';
+    id: string;
+    name: string;
+    email: string;
+    job_title: string;
+    away_mode_enabled: boolean;
+    away_mode_reassign: boolean;
+    has_inbox_seat: boolean;
+    team_ids: Array<number>;
+    avatar: string;
```

Comment:
why this change?

Reply:
Seems like linter was applied for this file for the first time :)